src/offset_of.rs

```diff
@@ -64,20 +54,77 @@
 /// assert_eq!(offset_of!(Vertex, loc), 0);
 /// assert_eq!(offset_of!(Vertex, color), 12);
 /// ```
+///
+/// # Usage with `#[repr(packed)]` structs
+///
+/// Attempting to compute the offset of a `#[repr(packed)]` struct with
+/// `bytemuck::offset_of!` requires an `unsafe` block. We hope to relax this in
+/// the future, but currently it is required to work around a soundness hole in
+/// Rust (See [rust-lang/rust#27060]).
+///
+/// [rust-lang/rust#27060]: https://github.com/rust-lang/rust/issues/27060
+///
+/// <p style="background:rgba(255,181,77,0.16);padding:0.75em;">
```

Comment:
Note: This is kind of gross looking in the docs, but looks very nice when rendered: https://imgur.com/a/blwzDfE